Game/Src/Objects/sword.gd

```diff
@@ -1,4 +1,34 @@
 extends KinematicBody2D
 
+onready var sprite = $swordsprite
+onready var normalTexture = load('res://Assets/Objects/Sword.png')
+onready var rusty1Texture = load('res://Assets/Objects/Sword-Rust1.png')
+onready var rusty2Texture = load('res://Assets/Objects/Sword-Rust2.png')
+onready var legendary1Texture = load('res://Assets/Objects/Sword-Epic1.png')
+onready var legendary2Texture = load('res://Assets/Objects/Sword-EpicMax.png')
 
+onready var max_glory = 100
 
+enum epic {normal,rusty,ruined,epic,legendary}
+
+func _ready():
+	sprite.set_texture(normalTexture)
+
+func _Glory_changed(value):
+	print('Sword Glory changed ',value)
+	if value <= 0.4*max_glory and value > 0.2*max_glory:
+		print('Rusty 1')
```

Comment:
here too I think the prints are no longer useful

Reply:
Yup, all the prints were for testing. Kept only to make sure the functions were entered.